cmd/zfs/zfs_main.c

```diff
@@ -306,7 +306,8 @@ get_usage(zfs_help_t idx)
 		    "<filesystem|volume|snapshot>\n"
 		    "\tsend [-DnPpvLec] [-i bookmark|snapshot] "
 		    "--redact <bookmark> <snapshot>\n"
-		    "\tsend [-nvPe] -t <receive_resume_token>\n"));
+		    "\tsend [-nvPe] -t <receive_resume_token>\n"
+		    "\tsend [-Pnv] -S filesystem\n"));
```

Comment:
It might be more "discoverable" to document it as `--saved` here, rather than `-S`.